src/lib/setupDatabaseHook.web.ts

```diff
@@ -1,11 +1,17 @@
 import type {QueryClient} from '@tanstack/react-query';
-import {db, type Row} from './db';
+import {db} from './db.web';
 import {databaseQueryHook} from './databaseQueryHook';
 
 export const setupDatabaseHook = (queryClient: QueryClient) => {
-  // TODO: this is a placeholder for the actual implementation that calls `databaseQueryHook()`
-  db.updateHook(params => {
-    const keyChanged = (params.row as Row).key;
+  db.kv.hook('creating', keyChanged => {
+    databaseQueryHook(keyChanged, queryClient);
```

Comment:
FYI @jbroma so far each hook is getting called correctly, but our `databaseQueryHook` somehow fails from time to time and doesn't feel reliable (on web only which makes it even more weird, it's the same, generic implementation).

Reply:
it's fixed already right?

Reply:
what about `Dexie.on.storagemutated` event, isn't that even simpler?